src/ctype/CType.spec.ts

```diff
@@ -53,6 +57,16 @@ describe('CType', () => {
     identityAlice.address
   )
 
+  it('compresses and decompresses the ctype object', () => {
+    const cType = CType.fromCType(fromCTypeModel)
+
+    const compressedCType = cType.compress()
+
+    expect(compressCType(cType)).toEqual(compressedCType)
```

Comment:
I am not sure if this test makes sense. This boils down to:

```js
expect(compressCType(cType)).toEqual(compressCType(cType))
```

because `cType.compress()` just called `compressCType(cType)`?

Reply:
Yes, the test are incomplete. 

Timo requested me to mock what the object needs to look like and what the arrays will have to look like as well. 

I haven't gotten round to it, as a lot is changing.